Makefile

```diff
@@ -1,11 +1,12 @@
-PREFIX=/usr/local
-BINDIR=$(DESTDIR)$(PREFIX)/bin
-MANDIR=$(DESTDIR)$(PREFIX)/share/man/man1
-DOCDIR=$(DESTDIR)$(PREFIX)/share/doc/googler
-UNAME_S:=$(shell uname -s)
+PREFIX ?= /usr/local
+BINDIR = $(DESTDIR)$(PREFIX)/bin
+MANDIR = $(DESTDIR)$(PREFIX)/share/man/man1
+DOCDIR = $(DESTDIR)$(PREFIX)/share/doc/googler
+BASHCOMPDIR = $(DESTDIR)$(PREFIX)/etc/bash_completion.d
+FISHCOMPDIR = $(DESTDIR)$(PREFIX)/share/fish/vendor_completions.d
+ZSHCOMPDIR = $(DESTDIR)$(PREFIX)/share/zsh/site-functions
```

Comment:
As for why not `/usr` and `/etc`, you can't simply assume you that have access to them, especially when you accept `PREFIX`. Also, shells are not necessarily installed in `/bin` or `/usr/bin` (my latest Bash, Fish and Zsh are all in `/usr/local`, for instance). I said "Please refer to your shell's manual for instructions on setting up your shell to use completions", so users should figure out paths themselves.